eventio/simtel/simtelfile.py

```diff
@@ -191,14 +200,21 @@ def next_low_level(self):
             for sub in o:
                 self.history.append(sub.parse())
 
+        elif isinstance(o, HistoryMeta):
+            if o.header.id == -1:
+                self.global_meta = o.parse()
+            else:
+                self.telescope_meta[o.header.id] = o.parse()
+
         elif isinstance(o, Histograms):
             self.histograms = o.parse()
         elif isinstance(o, iact.AtmosphericProfile):
             self.atmospheric_profiles.append(o.parse())
         else:
-            raise Exception(
+            warnings.warn(
                 'object type encountered, which is no handled'
-                'at the moment: {}'.format(o)
+                'at the moment: {}'.format(o),
```

Comment:
if you are either way changing this line you can also add a space, otherwise it gets joint as "handledat"

Reply:
Good catch, thanks